storybook/index.html

```diff
@@ -44,8 +53,48 @@ <h1 class="sectionHeading">
 
     <section class="demo">
       <div class="scrollCarousel">
-        <!-- .scrollCarousel demo goes here -->
-        (coming soon)
+        <div class="scrollCarousel--contents">
+          <div class="mover-1">ewefgs</div>
```

Comment:
looking pretty good now

like we said in class, it'll be better for demoing if we swap these divs for images

e.g. 

`<img src="https://static.wixstatic.com/media/d45a2e_9d25ee533e854c1ba4b3007f1789f6b4~mv2.jpg/v1/fill/w_416,h_324,al_c,q_80,usm_0.66_1.00_0.01/d45a2e_9d25ee533e854c1ba4b3007f1789f6b4~mv2.webp" width="200">`